deid/hipaa_dob_shift.sql

```diff
@@ -42,9 +42,26 @@ select
 from
     min_max_date_events cmm
 left join
-    "&&prev_cms_id_schema".min_max_date_events pmm
+    "&&prev_cms_id_schema"."&&mn_mx_dt_events_prev_yr_cumul" pmm

```

Comment:
Where is `mn_mx_dt_events_prev_yr_cumul` defined / documented?

Reply:
Documented the table definition with example.